home-application/src/main/java/com/softserveinc/ita/homeproject/application/exception/mapper/BadRequestExceptionMapper.java

```diff
@@ -13,9 +13,14 @@
  * @see javax.ws.rs.ext.ExceptionMapper
  */
 @Provider
-public class BadRequestHomeExceptionMapper extends BaseHomeExceptionMapper<BadRequestHomeException> {
+public class BadRequestExceptionMapper extends BaseExceptionMapper<BadRequestException> {
```

Comment:
I don't think it is a good idea to change it since it is in use for some other code. CC: @tex1988

Reply:
I still think this comment is actual.

Reply:
Unwrapper is added in BaseExceptionMapper